### PR TITLE
Remove `use_auth_token` from paddlenlp snippets

### PR DESCRIPTION
**Pull request by Wauplin**
Vaguely related to [slack thread](https://huggingface.slack.com/archives/C021H1P1HKR/p1701773657353989?thread_ts=1701719404.424999&cid=C021H1P1HKR) (private).

For paddlenlp, if a model is private, the code snippet will suggest to use `use_auth_token=True`. This is not needed anymore since the token is send on every request anyway now (since ~1y ago).

I have quickly reviewed paddlenlp source code and they seem to be correctly using `huggingface_hub` (so don't need `use_auth_token=True`). Cc @osanseviero @julien-c please let me know if I'm missing some context otherwise we can review/merge this PR.

### CODE REVIEW

**Review by osanseviero — COMMENTED** [review]
Thanks! I think we'll also need the minimum `huggingface_hub` version over there, right?

https://github.com/PaddlePaddle/PaddleNLP/blob/develop/requirements.txt#L11

cc @xianbaoqian for vis

**Review by julien-c — APPROVED** [review]
i agree with this change!

**Comment by Wauplin** [comment]
> Thanks! I think we'll also need the minimum huggingface_hub version over there, right?

@osanseviero  I just checked and the _"token is always sent by default when a user is logged in"_ rule has been introduced in [v0.10.0](https://github.com/huggingface/huggingface_hub/releases/tag/v0.10.0). So already enough for the paddlenlp minimum version (0.11.1) (though this is a very outdated version)

File: packages/tasks/src/library-ui-elements.ts [hunk 1]
```diff
@@ -187,12 +187,8 @@ const paddlenlp = (model: ModelData) => {
...
 				`from paddlenlp.transformers import AutoTokenizer, ${architecture}`,
 				"",
-				`tokenizer = AutoTokenizer.from_pretrained("${model.id}"${
-					model.private ? ", use_auth_token=True" : ""
-				}, from_hf_hub=True)`,
-				`model = ${architecture}.from_pretrained("${model.id}"${
-					model.private ? ", use_auth_token=True" : ""
-				}, from_hf_hub=True)`,
+				`tokenizer = AutoTokenizer.from_pretrained("${model.id}, from_hf_hub=True)`,
+				`model = ${architecture}.from_pretrained("${model.id}, from_hf_hub=True)`,
```
Comment (pcuenca) [inline packages/tasks/src/library-ui-elements.ts:191]:
```suggestion
				`tokenizer = AutoTokenizer.from_pretrained("${model.id}", from_hf_hub=True)`,
				`model = ${architecture}.from_pretrained("${model.id}", from_hf_hub=True)`,
```

File: packages/tasks/src/library-ui-elements.ts [hunk 2]
```diff
@@ -201,12 +197,8 @@ const paddlenlp = (model: ModelData) => {
...
 				`from paddlenlp.transformers import AutoTokenizer, AutoModel`,
 				"",
-				`tokenizer = AutoTokenizer.from_pretrained("${model.id}"${
-					model.private ? ", use_auth_token=True" : ""
-				}, from_hf_hub=True)`,
-				`model = AutoModel.from_pretrained("${model.id}"${
-					model.private ? ", use_auth_token=True" : ""
-				}, from_hf_hub=True)`,
+				`tokenizer = AutoTokenizer.from_pretrained("${model.id}, from_hf_hub=True)`,
+				`model = AutoModel.from_pretrained("${model.id}, from_hf_hub=True)`,
```
Comment (pcuenca) [inline packages/tasks/src/library-ui-elements.ts:201]:
```suggestion
				`tokenizer = AutoTokenizer.from_pretrained("${model.id}", from_hf_hub=True)`,
				`model = AutoModel.from_pretrained("${model.id}", from_hf_hub=True)`,
```

**Review by pcuenca — COMMENTED** [review]
Nice!

File: packages/tasks/src/library-ui-elements.ts (hunk 1 above)
Comment (Wauplin) [reply packages/tasks/src/library-ui-elements.ts:191]: Thanks for noticing @pcuenca !

**Review by merveenoyan — APPROVED** [review]
Looks good!

**Review by osanseviero — APPROVED** [review]
LGTM once the linting error is fixed :)

**Comment by Wauplin** [comment]
Thanks for the review everyone! Let's get this merged :)

**Pull request merged**